inspec.yml

```diff
@@ -0,0 +1,377 @@
+name: cms-ars-5.0-microsoft-windows-server-2016-stig-overlay
+title: cms-ars-5.0-microsoft-windows-server-2016-stig-overlay
+maintainer: CMS SAF Team
+copyright: 
+copyright_email: saf@cms.hhs.gov
+license: Apache-2.0
+summary: CMS ARS 5.0 Overlay InSpec Validation Profile for Windows Server 2016 STIG
+version: 1.7.0
+inspec_version: ">= 4.0"
+
+depends:
+  - name: microsoft-windows-server-2016-stig-baseline
+    url: https://github.com/mitre/microsoft-windows-server-2016-stig-baseline/archive/parameterize.tar.gz
+
+inputs:
+  - name: system_categorization
+    desc: "Selected system baseline based on the environment variable"
+    type: String
+    value: <%= ENV['BASELINE'].nil? ? 'Moderate (Default)' : ENV['BASELINE'] %>
+
+  - name: org_name
+    desc: "Name of the organization running this profile"
+    type: Hash
+    value:
+      acronym: "CMS"
+      full_form: "Centers for Medicare & Medicaid Services"
+    profile: microsoft-windows-server-2016-stig-baseline
+
+  - name: temporary_account_period # V-73283
+    desc: "List the number of days that temporary accounts remain active for"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'Moderate-HVA', 'Low-HVA'].include? ENV['BASELINE'] %>
+    value: 0.5
+    <% elsif ENV['BASELINE'] == 'High' %>
+    value: 30
+    <% elsif ENV['BASELINE'] == 'Moderate' %>
+    value: 60
+    <% elsif ENV['BASELINE'].nil? %>
+    value: 60
+    <% end %>
+
+  - name: temporary_account_period_phrase # V-73283
+    desc: "List the number of hours/days that temporary accounts remain active for"
+    type: String
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'Moderate-HVA', 'Low-HVA'].include? ENV['BASELINE'] %>
+    value: "12 hours"
+    <% elsif ENV['BASELINE'] == 'High' %>
+    value: "30 days"
+    <% elsif ENV['BASELINE'] == 'Moderate' %>
+    value: "60 days"
+    <% elsif ENV['BASELINE'].nil? %>
+    value: "60 days"
+    <% end %>
+
+  - name: emergency_account_period # V-73285
+    desc: "List the number of days that temporary accounts remain active for"
+    type: Numeric
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'Moderate-HVA', 'Low-HVA'].include? ENV['BASELINE'] %>
+    value: 0.5
+    <% elsif ['High', 'Moderate'].include? ENV['BASELINE'] %>
+    value: 1
+    <% elsif ENV['BASELINE'].nil? %>
+    value: 1
+    <% end %>
+
+  - name: emergency_account_period_phrase # V-73285
+    desc: "List the number of hours/days that temporary accounts remain active for"
+    type: String
+    profile: microsoft-windows-server-2016-stig-baseline
+    <% if ['High-HVA', 'Moderate-HVA' 'Low-HVA'].include? ENV['BASELINE'] %>
+    value: "12 hours"
+    <% elsif ['High', 'Moderate'].include? ENV['BASELINE'] %>
+    value: "24 hours"
+    <% elsif ENV['BASELINE'].nil? %>
+    value: "24 hours"
+    <% end %>
+
+  - name: pass_lock_duration # V-73309
```

Comment:
Figured out why this input is leading to the following error:
<img width="940" alt="Screen Shot 2023-06-09 at 9 55 26 AM" src="https://github.com/CMS-Enterprise/cms-ars-5.0-microsoft-windows-server-2016-stig-overlay/assets/11844975/cafd2c84-3fc7-4b2b-9003-590bcf4ebc39">

It needs to be assigned to a variable before being used in the `describe.one`

Reply:
added 
```
pass_lock_duration = input('pass_lock_duration') before describe.one
````